homeassistant/components/binary_sensor/hive.py

```diff
@@ -0,0 +1,66 @@
+"""
+Support for the Hive devices.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/hive/

```

Comment:
You should link to the page of the hive binary_sensor platform.

Reply:
ah, missed that. thanks.
done.